desktop/cmp/dash/canvas/DashCanvasModel.js

```diff
@@ -219,12 +223,12 @@ export class DashCanvasModel extends HoistModel {
      */
     @action
     removeView(id) {
-        const removeLayout = this.getLayout(id),
+        const removeLayout = this.getViewLayout(id),
             removeView = this.getView(id);
 
-        this.layouts = without(this.layouts, removeLayout);
```

Comment:
was this just a typo -- 'layouts' ?? Weird we did not see a bug here.

Reply:
I know- I think perhaps the reason we didn't see a bug is because the old viewModel was correctly being destroyed, so the layout state just contained obsolete data that didn't actually connect to an active view.  I'll take a look at dev currently and see if this is what's happening.